cli_tests/install_ketch.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+
+# exit when any command fails
+set -e
```

Comment:
how about also adding `set -o pipefail`? This setting prevents errors in a pipeline from being masked. If any command in a pipeline fails, that return code will be used as the return code of the whole pipeline. By default, the pipeline's return code is that of the last command even if it succeeds.

Reply:
Also please consider `set -x`. It enables a mode of the shell where all executed commands are printed to the terminal. In our case, it would be helpful for debugging, which is a typical use case for set -x printing every command as it is executed may help us to visualize the control flow of the script if it is not functioning as expected.

Reply:
Good call. Added `x` and `o`.